PaigeVannelli/shine
# Feature/router

## Pull Request
### Description
* Is this a feature or a fix?
Feature!

* Where should the reviewer start?
App renderComponent function and render, NewPostForm link

* What functionalities do these changes affect? Why were these changes made? What was the motivation behind these changes?
Adds router to application

* What outside features or research did you use to implement this fix?
N/A

* How should this be tested?
Please pull down the code and make sure you are able to navigate to and from the home page and new post form

closes #2
